### ISSUE TITLE
code fold indicator show strange characters

### ISSUE THREAD

**Issue by tjx666** [microsoft/vscode#146660]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/161433670-cd877fdf-6127-4b2b-80d3-6afd5386bd8c.png">

- VS Code Version: 1.67.0-insider
- OS Version: MacOS 12.3.1

**Comment by aeschli** [comment]
I suspect it's a Chromium rendering issue. We just use a `::after` css rule here:https://github.com/microsoft/vscode/blob/main/src/vs/editor/contrib/folding/browser/folding.css#L27

**Comment by tjx666** [comment]
I can't find a way to stably reproduce it. But I can sometimes occur it.

**Comment by tjx666** [comment]
> I remember someone in our team mentioning that the issue appears only after opening the Developer Tools. @tjx666 Can you confirm that you've been opening the developer tools maybe before this occurs?

Not for sure, I often open devtools today, but not sure weather I open it before this issue occurs.

**Comment by tjx666** [comment]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/169370832-23268ea0-192b-4c62-a5a3-ad309b4ac786.png">

@alexdima

**Comment by alexdima** [comment]
@tjx666 We believe this is a bug with Developer Tools. If you don't open Developer Tools, the `...` are rendered OK. If you open Developer Tools, sometimes, the `...` become garbled. The steps to reproduce is to open Developer Tools.

**Comment by aeschli** [comment]
duplicate of https://github.com/microsoft/vscode/issues/145897
